mpi-proxy-split/mpi-wrappers/mpi_request_wrappers.cpp

```diff
@@ -156,9 +156,14 @@ USER_DEFINED_WRAPPER(int, Testany, (int) count,
   MPI_Status *local_status = status;
 
   int retval;
+  bool all_requests_null = true;
```

Comment:
I think there is a simpler logic that doesn't need the variable `all_requests_null`.  I propose removing this variable, and making the changes noted below.

But if we must change the name, then this is better:
  `all_requests_null` -> `all_requests_are_null`